### DISCUSSION TITLE
4.10

### DISCUSSION THREAD

**Discussion by jrosdahl** [ccache/ccache#1462] [General]
* [Generic source code release (tar.gz)](https://github.com/ccache/ccache/releases/download/v4.10/ccache-4.10.tar.gz) ([signature](https://github.com/ccache/ccache/releases/download/v4.10/ccache-4.10.tar.gz.asc))
* [Generic source code release (tar.xz)](https://github.com/ccache/ccache/releases/download/v4.10/ccache-4.10.tar.xz) ([signature](https://github.com/ccache/ccache/releases/download/v4.10/ccache-4.10.tar.xz.asc))
* [Darwin (macOS) universal binary release](https://github.com/ccache/ccache/releases/download/v4.10/ccache-4.10-darwin.tar.gz) ([signature](https://github.com/ccache/ccache/releases/download/v4.10/ccache-4.10-darwin.tar.gz.asc))
* [Linux x86_64 binary release](https://github.com/ccache/ccache/releases/download/v4.10/ccache-4.10-linux-x86_64.tar.xz) ([signature](https://github.com/ccache/ccache/releases/download/v4.10/ccache-4.10-linux-x86_64.tar.xz.asc))
* [Windows i686 binary release](https://github.com/ccache/ccache/releases/download/v4.10/ccache-4.10-windows-i686.zip) ([signature](https://github.com/ccache/ccache/releases/download/v4.10/ccache-4.10-windows-i686.zip.asc))
* [Windows x86_64 binary release](https://github.com/ccache/ccache/releases/download/v4.10/ccache-4.10-windows-x86_64.zip) ([signature](https://github.com/ccache/ccache/releases/download/v4.10/ccache-4.10-windows-x86_64.zip.asc))
* [Release notes](https://ccache.dev/releasenotes.html#_ccache_4_10)
* [Manual](https://ccache.dev/manual/4.10.html)

<hr /><em>This discussion was created from the release <a href='https://github.com/ccache/ccache/releases/tag/v4.10'>4.10</a>.</em>

**Comment by silverqx** [comment]
Now I opened discussions that I will ask when you plan to release v4.10, nice. 🙃 Super happy about this release huge updates for msvc 🎉

**Reply by silverqx** [reply]
I'm already using 4.10 for all my dev. builds on msvc and msys2 and I can tell it's fire, game changer, it could be called 4.10 green release 😁 It already saved at least 1 tree 😅 Also, 0 bugs 👌
